modules/hcp-support-matrix.adoc

```diff
@@ -144,6 +144,139 @@ In the following table, the management cluster version is the {product-title} ve
 |4.16 - 4.18
 |===
 
+[id="hcp-matrix-multiarch_{context}"]
+== Multi-architecture support
+
+The following tables indicate the support status for {hcp} on multiple architectures, organized by platform.
+
+.Multi-architecture support for {hcp} on {aws-short}
+[cols="3",options="header"]
+|===
+|{product-title} version |ARM64 control planes |ARM64 worker nodes
+
+|4.18
+|General Availability
+|General Availability
+
+|4.17
+|General Availability
+|General Availability
+
+|4.16
+|Technology Preview
+|General Availability
+
+|4.15
+|Technology Preview
+|General Availability
+
+|4.14
+|Technology Preview
+|General Availability
+
+|4.13
```

Comment:
I think we should not mention 4.13 because it's out-of-life.
https://access.redhat.com/support/policy/updates/openshift